### ISSUE TITLE
Problem with DeepL extension

### ISSUE THREAD

**Issue by Filou53** [pilotmoon/PopClip-Extensions#1257]
Hello.
It seems to me that the DeepL Translator extension no longer works when using the DeepL Website mode.
Do others have the same problem?
Do you have a solution?
Thanks in advance...

**Comment by stefan786** [comment]
Yeah, I have the same issue, the extension no longer fills out the source text box. This seems to have been an unsupported hack?

**Comment by pilotmoon** [comment]
Yes, it looks like DeepL dropped support for supplying text to the web page via the URL. Whilst this always worked it  was never officially documented. I expect I will remove the DeepL Website option from the extension.
